mobify/platform-scaffold
# File size analyzer feedback

Addressing feedback from https://github.com/mobify/platform-scaffold/pull/746

## Changes
- A new `npm run test:build-size` that is separate from `lighthouse/check-score.js` and verifies that the contents of `build` are less than a specified threshold

## How to test-drive this PR
- Rebuild on https://circleci.com/gh/mobify/platform-scaffold/tree/file-size-analyzer-feedback
- Expand the 2nd container, expand the step `./tests/system/run-all-tests-on-ci.sh`
- Verify that the test runs after the unit tests

Locally
- Change the `file_size_limit` value in `web/package.json` then 
```
npm run prod:build
npm run test:build-size
```
